keyboards/consolekeyboard/readme.md

```diff
@@ -0,0 +1,15 @@
+# Console Keyboard
+
+[Console Keyboard](https://i.imgur.com/5aLT7CL.jpeg)
+
+A 3d printed macro pad based on the Stream Cheap
```

Comment:
Is this a handwired keyboard, or does it use a PCB?

Reply:
its hand wired

Reply:
Then the files should be moved to the `keyboards/handwired/` folder